pkg/state/state.go

```diff
@@ -2563,6 +2571,8 @@ func (st *HelmState) flagsForUpgrade(helm helmexec.Interface, release *ReleaseSp
 
 	flags = st.appendHelmXFlags(flags, release)
 
+	flags = st.appendLabelsFlags(flags, helm, release)
```

Comment:
The release label in helmfile only allows users to choose release. Is it necessary to add it to helm installation parameters?

Reply:
see: https://github.com/helm/helm/pull/10533